### ISSUE TITLE
No Ports icon!

### ISSUE THREAD

**Issue by haydenbjyoung** [lilbud/es-theme-switch-light#2]
I absolutely adore this theme for ES, but I have found a couple problems with it.

First, there is no Ports icon, so it just shows up as `ports` in big lettering, which is slightly annoying. And secondly, the background makes the colour of the icons a bit hard to see. If you could change the main ribbon colour of EmulationStation to grey or something, that'd be great.

Hope to see more updates to this soon!

**Comment by lilbud** [comment]
I keep forgetting to make a ports icon, I'll get on it soon.

And about the ribbon color, there should be none unless you haven't updated ES. I designed the theme with no ribbon in the middle. So try updating.

**Comment by haydenbjyoung** [comment]
I must say, that might just be the fastest reply I've ever had to a GitHub issue, lol!

And is there an option for updating it in `retropie-setup`? I'm assuming there should be.

**Comment by haydenbjyoung** [comment]
Yup, there was. I updated from binary but just to be sure, I'm gonna update it from the source code. Thanks for that!

**Comment by haydenbjyoung** [comment]
@lilbud Should this also fix the error I've been having where I don't get *any* icons or pictures on your `nes-mini` theme?

**Comment by lilbud** [comment]
Umm, the nes-mini theme isn't mine, that belongs to Ruckage. But both mine and his theme support a custom carousel, which is only supported on the newest emulationstation. Update from source and tell me how that goes.

**Comment by haydenbjyoung** [comment]
Alright, cheers. Updating from the latest GH release worked, so I have the theme working. Thanks for the help! I'll have a look at fixing the `nes-mini` theme, I was unaware that it wasn't maintained by you.
